docs/versioning-policy.md

```diff
@@ -0,0 +1,7 @@
+# Versioning Policy
```

Comment:
Maybe call this file `Versioning-Policy.md` to match the other names in `docs/`?